test/authextension.test.ts

```diff
@@ -53,6 +53,8 @@ suite('authextension Modules', () => {
       context_f8_access_routes,
       context_f8_3scale_user_key
     );
+    process.env['RECOMMENDER_API_URL'] = 'http://prod/api/v2';
+    process.env['THREE_SCALE_USER_TOKEN'] = '12345';
```

Comment:
@dvandra rest all changes look good, but this is where i am not very sure. 
May be @arajkumar can tells us more on this.
Overall we will be anyways reverting the PR hence may be we can go with this.